app/controllers/recycle_location_changes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RecycleLocationChangesController < ApplicationController
+  def create
+    # Make change variable that holds new data
+    change = RecycleLocationChange.new
+    change.import_from_params(params)
+
+    # Save change to database
+    change.save
```

Comment:
You may want to check if this `RecycleLocationChange ` instance is a valid instance and can be saved before you return `:ok` 

You can do something like --> 
```
if change.save
  head :ok
else ...
```